React/Views/RCTViewKeyboardEvent.h

```diff
@@ -0,0 +1,14 @@
+/*
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#import <Foundation/Foundation.h>
+#import <React/RCTComponentEvent.h>
+
+@interface RCTViewKeyboardEvent : RCTComponentEvent
+
++ (instancetype)keyDownEventWithReactTag:(NSNumber *)reactTag characters:(NSString*)characters modifier:(NSUInteger)modifier;
```

Comment:
Superduperminor nit pick

```suggestion
+ (instancetype)keyDownEventWithReactTag:(NSNumber *)reactTag characters:(NSString *)characters modifier:(NSUInteger)modifier;
```

We should really get some alignment with upstream about rigorously running clang-format to also ease merges.

Reply:
Agree that'll be nice. I searched around a few examples in the existing rn code and saw spaces between the class name and star so just followed that. I wouldn't be surprised if it's inconsistent though